app/src/main/java/nl/tudelft/b_b_w/controller/BlockController.java

```diff
@@ -158,10 +159,16 @@ public final int getLatestSeqNumber(String owner) {
     @Override
     public final List<Block> revokeBlock(Block block) {
         String owner = block.getOwner();
-        Block newBlock = BlockFactory.getBlock(REVOKE, block.getOwner(),
-                block.getOwnHash(), block.getPreviousHashChain(), block.getPreviousHashSender(),
-                block.getPublicKey(), block.getIban(), block.getTrustValue());
-        addBlock(newBlock);
+        addBlock(BlockFactory.getBlock(
+                REVOKE,
+                owner,
```

Comment:
Because otherwise you make two get calls to the block object, while you already have the owner attribute.